modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
-+
-image::sending-data-from-arm-to-external-monitoring-software-level.png[]
+[WARNING]
+If you configure the API gateway `wrapper.conf` (step 1) but don't assign an external destination for your data (step 2), the analytics data accumulates in a queue in the API gateway server, which could cause the system to crash.
 
-. On the *Event Tracking* region, activate the *ELK* switch. This will open a pop up menu where you can provide the address to the folder where you keep the log files that your ELK stack reads.
-+
-image::sending-data-from-arm-to-external-monitoring-software-elk.png[]
 
-. Optionally, you can open the advanced menu and set up certain formatting properties of the data you send out and how the information is archived.
-+
-image::elk-config-advanced.jpg[ELK advanced]
+=== Configure the API Gateway
 
-=== Configure API Analytics
-
-Before you can set up the connection to external software through the Runtime Manager UI, you must first make some changes to the API gateway to prepare it for this.
+To set up the connection to external analytics tools, configure the API gateway:
 
 . In your API gateway standalone directory, open the `conf/wrapper.conf` file.
++
+The `.<n>` in the code snippets refers to an integer that is unique within `wrapper.conf`.
 . Ensure that the following property is set to `true` and that the line is uncommented:
 +
 [source,java,linenums]
 ----
 wrapper.java.additional.<n>=-Danypoint.platform.analytics_enabled=true
 ----
 
-. Look for this other line:
+. Remove the URL from the following line:
 +
 [source,java,linenums]
 ----
 wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.anypoint.mulesoft.com
 ----
-. Remove the URL in it, so that it looks like this:
++
+The line now looks like the following:
 +
 [source,java,linenums]
 ----
 wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=
 ----
-. When using Anypoint Platform Private Cloud Edition, there's one more parameter you need to change:
+. For Anypoint Platform Private Cloud Edition, change the `anypoint.platform.on_prem` parameter to `true`:
 +
 [source,java,linenums]
 ----
 wrapper.java.additional.<n>=-Danypoint.platform.on_prem=true
 ----
 +
-`anypoint.platform.on_prem` is set to `false` by default. To manage it through Anypoint Platform Private Cloud Edition, you must set it to `true`. To manage it through the Runtime Manager in the cloud, leave it as `false`.
+If you are using the Runtime Manager cloud console, leave this parameter set to the default value, `false`. 
 
-[TIP]
-Note that in the above code snippets, when lines that contain `.<n>`, that should be replaced with an integer number that is unique within the wrapper.
 
-Once you have set up your Stand Alone API gateway, the steps for connecting to Splunk and ELK are identical to those when dealing with Mule Custom Events, except that you should set them up via the corresponding switches.
+==== Configure Multiple Consumers 
```

Comment:
At some point when you have time-HA!- you might look to find a way to avoid four levels of information nesting.